src/styles/SectionQuestions.less

```diff
@@ -0,0 +1,43 @@
+.container {
+    border: solid 1px black;
```

Comment:
Use color variables in separate file?

Reply:
Legger til color.less nå slik det er gjort i repoet til rodekors.